bugsnag-plugin-android-ndk/src/main/jni/utils/serializer.c

```diff
@@ -228,24 +229,23 @@ void bsg_serialize_device(const bsg_device_info device, JSON_Object *event) {
   }
 
   json_object_dotset_number(event, "device.totalMemory", device.total_memory);
+
+  char report_time[sizeof "2018-10-08T12:07:09Z"];
```

Comment:
I don't think this should be part of this PR – it doesn't seem related to removing obsolete `notify()` calls?

Reply:
It appears that this also contains the smaller changeset for #645, so this should disappear from the diff once that is merged.